mslib/mscolab/seed.py

```diff
@@ -218,7 +218,10 @@ def archive_operation(path=None, emailid=None):
                 elif perm.access_level != "creator":
                     return False
                 operation.active = False
-                operation.last_used = datetime.datetime.utcnow() - dateutil.relativedelta.relativedelta(months=2)
+                operation.last_used = (
```

Comment:
add here a 
# ToDo use mscolab_settings.ARCHIVE_THRESHOLD instead of months=2

This remembers us, or will become a next issue. There may be more places. 
A change of this is in the current PR is not a good idea 

When changing something we have always to have in mind we could rollback, just one thing. But when things are mixed up it gets difficult.

In this case a #ToDo is the solution

Reply:
I file a bug report for stable.

Reply:
Now you don't need to add the ToDo's